ee/ucp/kubernetes/storage/use-azure-disk.md

```diff
@@ -0,0 +1,237 @@
+
+
+## Pre-Requisites
+
+This guide assumes you have already provisioned a UCP environment on to
+Microsoft Azure. The Cluster must be provisioned after meeting all of the
+pre-requisites within [Install UCP on
+Azure](/ee/ucp/admin/install/install-on-azure.md).
+
+Additionally the following guide is using the Kubernetes Command Line tool `$
+kubectl` to provision Kubernetes objects within a UCP cluster. Therefore this
+tool needs to be downloaded, along with a UCP client bundle. For more
+information on configuring CLI access to UCP see [CLI Based
+Access](/ee/ucp/user-access/cli/).
+
+## Manually Provisioning Azure Disks
+
+An operator can use existing Azure Disks or manually provision new ones to
+provide persistent storage for Kubernetes Pods. Azure Disks can be manually
+provisioned in the Azure Portal, using ARM Templates or using the Azure Cli. In
+the below example we have used the Azure Cli to manually provision an Azure
+Disk. 
+
+```
+$ RG=myresourcegroup
+
+$ az disk create \
+  --resource-group $RG \
+  --name k8s_volume_1  \
+  --size-gb 20 \
+  --query id --output tsv
+```
+
+Using the above Azure CLI command should return the Azure ID of the Azure Disk
+Object. If you are provisioning Azure resources using an alternative method,
+make sure you retrieve the Azure ID of the Azure Disk as you will need it later
+on. 
+
+```
+/subscriptions/<subscriptionID>/resourceGroups/<resourcegroup>/providers/Microsoft.Compute/disks/<diskname>
+```
+
+You can now create Kubernetes Objects referring to this Azure Disk. The below
+example is using a Kubernetes Pod, however the same Azure Disk syntax can be
+used for Daemonsets, Deployments and Statefulsets. In the below example the
+Azure Disk Name and ID to reflect the manually created Azure Disk. 
+
+```
+$ cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mypod-azuredisk
+spec:
+  containers:
+  - image: nginx
+    name: mypod
+    volumeMounts:
+      - name: mystorage
+        mountPath: /data
+  volumes:
+      - name: mystorage
+        azureDisk:
+          kind: Managed
+          diskName: k8s_volume_1
+          diskURI: /subscriptions/<subscriptionID>/resourceGroups/<resourcegroup>/providers/Microsoft.Compute/disks/<diskname>
+EOF
+```
+
+## Dynamically Provisioning Azure Disks
+
+### Defining the Azure Disk Storage Class
+
+Kubernetes can dynamically provision Azure Disks using the Azure Kubernetes
+integration which was configured for you when UCP was installed. For Kubernetes
+to know which APIs to use when provisioning storage, Operators need to create
+Kubernetes Storage Classes specific to each storage backend. For more
+information on Kubernetes Storage Class, see [Storage
+Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/).
+
+In Azure there are 2 different types of Azure Disk that can be consumed by
+Kubernetes. Azure Disk Standard Volumes and Azure Disk Premium Volumes. For more
```

Comment:
A `:` after `In Azure there are 2 different types of Azure Disk that can be consumed by
Kubernetes:` would setup the two options